packages/dotcom-ui-header/package.json

```diff
@@ -23,7 +23,8 @@
   "license": "MIT",
   "dependencies": {
     "@financial-times/dotcom-types-navigation": "file:../dotcom-types-navigation",
-    "n-topic-search": "^8.0.0"
+    "n-topic-search": "^8.0.0",
+    "preact": "^10.23.2"
```

Comment:
**issue:** We should add this as a peer dependency so that a consuming system will use a version of preact that matches the version range (if it already has a version.)

Reply:
changed!

Reply:
finally the class was not need and the close text should be shown